workspace/TS100/Core/BSP/Miniware/Setup.c

```diff
@@ -337,7 +337,7 @@ static void MX_TIM2_Init(void) {
 	HAL_TIMEx_MasterConfigSynchronization(&htim2, &sMasterConfig);
 
 	sConfigOC.OCMode = TIM_OCMODE_PWM1;
-	sConfigOC.Pulse = 255 + 13;  //13 -> Delay of 5ms
+	sConfigOC.Pulse = 255 + 13 * 2;  // 13 -> Delay of 7 ms
```

Comment:
Why `13*2`?

Reply:
Since I double the frequency I need the ~double amount of ticks to get the same delay.